plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClientImpl.kt

```diff
@@ -400,6 +401,17 @@ class AmazonQLanguageClientImpl(private val project: Project) : AmazonQLanguageC
         return CompletableFuture.completedFuture(Unit)
     }
 
+    override fun chatOptionsUpdate(params: LSPAny): CompletableFuture<Unit> {
```

Comment:
nit: should we call it `sendChatOptionsUpdate` to be consistent with `sendChatUpdate`?